### DISCUSSION TITLE
migrating from v2 to v3

### DISCUSSION THREAD

**Discussion by rohitjswr** [gatsbyjs/gatsby#36564] [Help]
Was trying to update my gatsby from v2 to v3
after a clean install, on gatsby develop getting this error

<img width="595" alt="image" src="https://user-images.githubusercontent.com/102286698/188927658-efc7a22c-8793-41bd-bcca-38df2d7629f3.png">

anyone aware about this

**Comment by marvinjude** [comment]
Hi @rohitjswr,

Sorry you're running into this. 

I'd advice that you follow migration guide [migration guide](https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/). If you're stilling running into this we'd very much appreciate if you could [provide a minimal reproduction](https://gatsby.dev/reproduction) and we'll be able to take a look. 

Thanks for using Gatsby! 💜
